profiler/src/main/java/com/splunk/opentelemetry/profiler/events/ContextAttached.java

```diff
@@ -22,10 +22,12 @@
 import jdk.jfr.Event;
 import jdk.jfr.Label;
 import jdk.jfr.Name;
+import jdk.jfr.StackTrace;
 
 @Name(EVENT_NAME)
 @Label("otel context attached")
 @Category("opentelemetry")
+@StackTrace(false)
```

Comment:
without the stack, why are we creating the event at all? I feel like I'm confused about what this is used for.

Reply:
It's used to know what thread is working on a span at a given instant in time, so that a stack from another event (like `jdk.ThreadDump`) can be linked to a span.

Reply:
👍🏼